public/js/p3/store/PathwaySummaryMemoryStore.js

```diff
@@ -173,7 +173,7 @@ define([
           return p + '=' + query[p];
         }).join('&');
 
-        return when(request.post(_self.apiServer + '/pathway/', {
+        return when(request.post(_self.apiServer + 'pathway/', {
```

Comment:
@JacobPorter Can you put these slashes back in for consistency?  It doesn't look like there's a trailing slash in the configs on the servers.

Reply:
The apiServer string ended with a slash, so that is why it was removed.  I can put it back in if you like since it won't hurt anything, but then there will be two slashes.